ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -265,7 +276,17 @@ QObject {
 
         sdk: root.wcSDK
         store: root.store
-        supportedAccountsModel: d.supportedAccountsModel
+        supportedAccountsModel: SortFilterProxyModel {
+            objectName: "SelectedAddressModelForDAppsListProvider"
+            sourceModel: d.supportedAccountsModel
+            filters: ValueFilter {
+                roleName: "address"
+                value: root.walletRootStore.selectedAddress === model.address
```

Comment:
WRN 2024-08-01 11:19:17.779+03:00 qt warning                                 topics="qt" tid=4930042 text="ReferenceError: model is not defined" file=qrc:/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml:284 category=default

Reply:
The value filter expect a value, like` root.walletRootStore.selectedAddress`. Although this is not 100% safe because we need the filter not to be case sensitive.
An alternative would be a FastExpressionFilter comparing the strings and ignoring the case

Reply:
Yeah, on a second look, I think it's safer to do a `FastExpressionFilter` unless we want to fiddle around with a `RegExpFilter` and setting the case sensitivity

Reply:
To be reverted back to `FastExpressionFilter`